packages/packages/v1-adapters/src/dispatchers/dispatchers.ts

```diff
@@ -0,0 +1,38 @@
+import { ExtendedWindow } from './types';
+import { isJQueryDeferred, promisifyJQueryDeferred } from './utils';
+
```

Comment:
maybe?

```ts
declare global {
	interface Window {
		$e?: {
			run?: ( command: string, args?: object ) => unknown;
			route?: ( route: string ) => void;
		},
	}
}
```

Reply:
it will make this type globally available to every package that imports this adapter, and we don't want that.